pages/common/laravel.md

```diff
@@ -0,0 +1,19 @@
+# laravel
+
+> A command line installer for the Laravel framework.
+
+- Create a new Laravel application:
+
+`laravel new {{name}}`
+
+- Create a new Laravel application even if the directory already exists:
+
+`laravel new {{name}} --force`
+
+- Create a new Laravel application with the latest development release:
```

Comment:
No need to say "Create a new Laravel application" after the second time. Perhaps change this to `Use the latest development release` ? Or is it too short ?